Dockerfile-slim.template

```diff
@@ -1,6 +1,7 @@
 FROM buildpack-deps:jessie-curl
 
-RUN groupadd -r node && useradd -r -g node node
+RUN groupadd --gid 1000 node \
+  && useradd --uid 1000 --gid node --create-home node
```

Comment:
Are you worried that the underlying `buildpack-deps` image might change and add a regular user with UID/GID 1000? The chance of that happening is negligible. Also, there was no similar check so far, why introduce it now?